pages/programs/youth-managed-farms.css

```diff
@@ -0,0 +1,168 @@
+/* Importing stuff! */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+
+/*Now the actual code */
+
+/* Grid Items */
+.grid-st {
+
+}
+.grid-mid {
+
+}
+.grid-end {
+	grid-column-start:2;
+	grid-column-end: 5;
+}
+/* Header Items */
+
+#headerBR {
+	display: grid;
+  	grid-template-columns: auto auto auto auto;
+	padding: 10px;
+	width:  75%;
+	margin: auto 15% auto 20%;
+	background-color: white;
+
+}
+
+#navBR {
+	margin-left: 20%;
+	display: flex;
+	white-space:nowrap;
+
+}
+/* nav Animation */
+#navBR a {
+	text-decoration: none;
+   font-family: 'Open Sans', sans-serif;
+	display: inline-block;
+	color: #000000;
+}
+.u-BR {
+	background-color: #000000;
+	height: 2px;
+	width: 100%;
+	display:block;
+  -webkit-transform: scaleX(0);
+	transform: scaleX(0);
+  -webkit-transition: all 0.25s;
+	transition: all 0.25s;
+}
+#navBR a:hover .u-BR {
+	-webkit-transform: scaleX(1);
+	transform: scaleX(1);
+}
+/*End nav Animation*/
+
+.navBR-1 {
```

Comment:
Don't repeat yourself. We can combine the selectors that share the same padding value and have a one off rule the item that has the height value